index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+
+<html>
+  <head>
+    <meta charset="UTF-8">
```

Comment:
On the same note, this is lowercase utf-8. I'm going off a generic website dom (this page works)